definitions/npm/storybook-router_v0.x.x/flow_v0.25.x-/storybook-router_v0.x.x.js

```diff
@@ -0,0 +1,37 @@
+import type { Element } from "react";
```

Comment:
Isn't ` import type { Element } from "react"; ` only available from Flow >= 0.53?
In such case Element would resolve `any` for all other lower versions.

Reply:
Correct, I figured it's not a huge deal if it does resolve to `any`. If you're keen to fix it, a PR would be much appreciated!

Reply:
Your right, that's fine as is I guess. ;)

Reply:
Thanks for all the hard work!